sensor/common/clusterentities/entity_status.go

```diff
@@ -0,0 +1,37 @@
+package clusterentities
+
+func newEntityStatus(numTicks uint16) *entityStatus {
+	return &entityStatus{
+		ticksLeft:    numTicks,
+		isHistorical: false,
+	}
+}
+
+type entityStatus struct {
+	ticksLeft    uint16
+	isHistorical bool
+}
+
+// markHistorical is called when entity would be deleted.
+// Istead, we mark it as historcal and keep it as long as ticksLeft
```

Comment:
```suggestion
// markHistorical is called when entity is deleted from the cluster.
// Instead of deleting it from memory, we mark it as historical and keep it as long as ticksLeft
```

Reply:
I totally missed this one - addressed in https://github.com/stackrox/stackrox/pull/9154